stable/kong/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.5.0
+version: 0.5.1
```

Comment:
Let's bump this up to 0.6.0 since there are other PRs on this charts which might cause conflicts and I want to minimize back and forth on this PR now.